doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -359,6 +359,7 @@ Peripheral samples
 * :ref:`radio_test` sample:
 
    * Added support for the nRF54LM20 PDK.
+   * Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` and  ``nrf54lm20pdk/nrf54lm20a/cpuapp`` boards target.
```

Comment:
```suggestion
      * Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` boards target.
```
The new agreement from the project management is to make sure the target (nRF54LM20 PDK) is not mentioned anywhere in the documentation until the preview-release in September. Therefore, took it out.